Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,250 @@
+```
+This will save the contents of the key in String format within the file found by following the specified path in .pem files.
+
+### Saving a public Key to RDF
+
+A user can also transform his public keys to an RDF. In order to do this one must first import multiple files in order to resolve the required dependencies.
+
+The org.w3 declarations required are:
+
+```scala
+import org.w3.banana.binder
+import org.w3.banana.binder.RecordBinder
+implicit val binder = RecordBinder
+val cert = CertPrefix[Rdf]
+```
+Several java imports from java's security and math library are also required for this process:
+
+```scala
+import java.math.BigInteger
+import java.security.KeyFactory
+import java.security.interfaces.RSAPublicKey
+import java.security.spec.RSAPublicKeySpec
+```
+
+After that the binder and all it's functionalities can be imported:
+
+```scala
+import org.w3.banana.binder._
+import recordBinder._
+```
+
+Finally, after that, the user can create the Cert object that contains the required dependencies to turn a Key into a Pointed Graph
+
+```scala
+object Cert {
+
+          implicit val rsaClassUri = classUrisFor[RSAPublicKey](cert.RSAPublicKey)
+          val factory = KeyFactory.getInstance("RSA")
+          val exponent = property[BigInteger](cert.exponent)
+          val modulus = property[Array[Byte]](cert.modulus)
+
+          implicit val binder: PGBinder[Rdf, RSAPublicKey] =
+          pgb[RSAPublicKey](modulus, exponent)(
+            (m, e) => factory.generatePublic(new RSAPublicKeySpec(new BigInteger(m), e)).asInstanceOf[RSAPublicKey],
+            key => Some((key.getModulus.toByteArray, key.getPublicExponent))
+            ) // withClasses rsaClassUri
+
+      }
+
+defined object Cert
+
+import Cert._
+```
+
+After this all dependencies should be resolved and the user will be able to transform the keys into a Pointed Graph:
+
+```scala
+@ val keyGraph = pub.toPG
+
+keyGraph: PointedGraph[RDF] = org.w3.banana.PointedGraph$$anon$1@7f8fa63d
+```
+One can then retrieve both the pointer and the graph:
+
+```scala
+@ keyPG = keyGraph.graph
+
+keyPG: RDF#Graph =  {5880.026289.0-02638.0-6288.0-06411.0-23110.021178.029228.0 http://www.w3.org/ns/auth/cert#exponent "65537"^^http://www.w3.org/2001/XMLSchema#integer; 5880.026289.0-02638.0-6288.0-06411.0-23110.021178.029228.0 @http://www.w3.org/ns/auth/cert#modulus "00937adccd722bc982aed4847872b81e36b890bca13166714bc2befe4d8547b6218ecd2da1eb020198a4ea00e4db6757c7dda738ec8db8b3bf211d3a3a17e196a2035bc4c79d06d8a581487d9f49e86374712b10ef500dfa242a20cab52911e2636c9d99b21fe9768ef2381989a25dc8b0b7a46531249aac27c4b8ab451a19d5fbdfa5f78b0deac9778c7ff87cf6106ae4a6433466beb21df1265bf1fc9ab9cc80d7aff8cc4d0f67ae28647e4048da8df753493b8de6a8e0961416b4b37f7012907d2e756034b8a84c6e495c8f1d81f69843ae51379571d83b38e1c08a08c3748ef75ed7ecb016d0c8426b30c8c5060a08f87f6764b0ec14667cd6f0daa1244157"^^http://www.w3.org/2001/XMLSchema#hexBinary}
+
+keyRDFPointer = keyGraph.pointer
```

Comment:
here you need both the `@` and to set `val =` since you are looking to see the results of running the command.